litmus-portal/frontend/src/pages/Settings/styles.ts

```diff
@@ -2,14 +2,15 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   root: {
+    color: 'inherit',
     display: 'flex',
     flexGrow: 1,
     maxWidth: '63.75rem',
     alignItems: 'center',
     justifyContent: 'space-between',
     marginLeft: theme.spacing(2),
     borderBottom: `1px solid ${theme.palette.customColors.black(0.1)}`,
-    backgroundColor: theme.palette.tabSelectorBackgroundColor,
+    backgroundColor: `${theme.palette.customColors.black(0)} !important`,
```

Comment:
please avoid using color with their name

Reply:
yes, will update it, thanks